src/NuGet.Core/NuGet.LibraryModel/LibraryIncludeFlagUtils.cs

```diff
@@ -24,6 +24,14 @@ public static class LibraryIncludeFlagUtils
         /// Convert set of flag strings into a LibraryIncludeFlags.
         /// </summary>
         public static LibraryIncludeFlags GetFlags(IEnumerable<string> flags)
+        {
+            return GetFlags(flags, includeBuildWithBuildTransitive: true);
+        }
+
+        /// <summary>
+        /// Convert set of flag strings into a LibraryIncludeFlags.
+        /// </summary>
+        public static LibraryIncludeFlags GetFlags(IEnumerable<string> flags, bool includeBuildWithBuildTransitive)
```

Comment:
There's still an API discussion, so I'm not expecting them added immediately, but given that this is a new public method, and it's super easy to test, we msut add unit tests for this.